Packs/HPE_Service_Manager/Integrations/integration-HPE_Service_Manager.yml

```
@@ -68,13 +62,11 @@ script:
             break;
         default:
     }
-
     return result;
 
-
-    function createIncident() {
+    function updateIncident() {
         var newIncident = {
-            Incident: {
+            [args.resourcename]: {
```

```suggestion
            var resourceName = args.resourcename
            [resourceName]: {
```
get the value from resourcename arg to variable and use it